packages/client/components/InsightsDomainPanel.tsx

```diff
@@ -147,10 +198,24 @@ const InsightsDomainPanel = (props: Props) => {
     activeOrganizationCount,
     activeTeamCount,
     activeUserCount,
-    meetingCount
+    meetingCount,
+    organizations
   } = domain
-
+  const toBeLockedOrg = organizations.find((org) => org.scheduledLockAt)
   const isDesktop = useBreakpoint(Breakpoint.NEW_MEETING_SELECTOR)
+  const atmosphere = useAtmosphere()
+  const {history} = useRouter()
+
+  const handleClick = () => {
+    if (!toBeLockedOrg) return
+    const {id: orgId} = toBeLockedOrg
+    SendClientSegmentEventMutation(atmosphere, 'Upgrade CTA Clicked', {
+      upgradeCTALocation: 'usageStats',
+      orgId
+    })
```

Comment:
~~+1 Can we tweak this to match the Meeting History (#7511) pattern?~~

```suggestion
    SendClientSegmentEventMutation(atmosphere, 'Upgrade Intent', {
      source: 'usageStats',
      upgradeTier: 'pro',
      orgId
    })
```

Reply:
Actually, upon further review I think it's better that we keep your `Upgrade CTA Clicked` event.  We already have logged some of these in production, and haven't registered any `Upgrade Intent` events yet, so it's probably cleaner to unify it under `Upgrade CTA Clicked`.

However, I would like to add in the `upgradeTier` parameter, as it'll help us as we expand our plan offerings in the future.

You've also done some nice work formalizing it under analytics.ts, thanks!  I'll put together a PR to migrate the events introduced by #7511 to match this pattern.

Reply:
Sure thing - I've added `upgradeTier` 👍